tests/testsuite/elixir.rs

```diff
@@ -0,0 +1,54 @@
+use ansi_term::Color;
+use std::fs::File;
+use std::io;
+
+use crate::common;
+
+#[test]
+fn folder_without_elixir_files() -> io::Result<()> {
```

Comment:
Missing an ignore here

Reply:
Ok. Should I add one to the similar test in ruby.rs as well?

Reply:
On second thought, I think the reason the test wasn't ignored is because it's expecting not to identify the path as being a project for the language (and therefore tests won't break on versions).